core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneticAlteration.java

```diff
@@ -77,6 +78,15 @@ public static DaoGeneticAlteration getInstance() throws DaoException {
         return daoGeneticAlteration;
     }
 
+    public static interface AlterationProcesser {
+        ObjectNode process(
+            long entrezGeneId,
+            String[] values,
+            ArrayList<Integer> orderedSampleList,
+            HashMap mutHm
```

Comment:
mutation hashmap --> result from calling DaoMutation.getSimplifiedMutations, instead querying for database